src/hooks/useIsDefaultToken.ts

```diff
@@ -0,0 +1,11 @@
+import { Token } from '@uniswap/sdk'
+
+import { ALL_TOKENS } from '../constants/tokens'
+
+export function isDefaultToken(token?: Token): boolean {
+  return Boolean(token && ALL_TOKENS[token.chainId]?.[token.address])
+}
+
+export default function useIsDefaultToken(token?: Token): boolean {
```

Comment:
I don't love wrapping this in a hook, it makes it seem like you have to call it in order (like with other hooks)